docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -94,6 +94,12 @@ In general, you shouldn't stop more than one node at a time. For example, two st
 
 You can restart the node's processes by navigating to **Universes**, selecting your universe, then selecting **Nodes**, and then clicking **Actions > Start Processes** corresponding to the node. This returns the node to the Live state.
 
+{{< warning title="Prevent back up failure due to NFS unmount on cloud VM restart" >}}
+When a cloud VM is restarted, the NFS mount may get unmounted if its entry is not present in `/etc/fstab`. This configuration oversight can lead to back up failures, causing errors during [backup](../../back-up-restore-universes/back-up-universe-data/) or [restoration](../../back-up-restore-universes/restore-universe-data/) processes.
```

Comment:
```suggestion
To avoid potential backup and restore errors, make sure the NFS mount is added to `/etc/fstab`. When a cloud VM is restarted, the NFS mount may get unmounted if its entry is in `/etc/fstab`. This can lead to backup failures, and errors during [backup](../../back-up-restore-universes/back-up-universe-data/) or [restore](../../back-up-restore-universes/restore-universe-data/).
```

Reply:
NFS config page doesn't say anything about this
https://docs.yugabyte.com/preview/yugabyte-platform/back-up-restore-universes/configure-backup-storage/#network-file-system
When creating NFS storage config, is this entry supposed to be added automatically? And if so why would it disappear?

Reply:
cc @kv83821-yb @zlareb1-yb

Reply:
NFS volumes are mounted on all the VMs. This mount could go away if the VM is restarted unless the mount is added to `/etc/fstab`. This should be done in order to make the volume auto-remount when the VM is restarted.

For Dwight's question: is this entry supposed to be added automatically ?
No, this is something that the user has to take care of on all the VMs.

Reply:
@kv83821-yb In YBA, I add NFS backup config via the UI as described in my link. At this point, I'm expected to tunnel to each node and manually add the mount to /etc/fstab/? This seems impractical.

Reply:
@kv83821-yb I see adding to fstab is included in manual provisioning
https://docs.yugabyte.com/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual/#pre-provision-nodes-manually
But should we mention it when creating NFS backup config